File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.java
class Solution {
    static int arr[] = new int[100001];
    public int solution(int n) {
        int answer = 0;
        arr[0]=0;
        arr[1]=1;
        for(int i=2; i<=n; i++){
            arr[i]=(arr[i-1]+arr[i-2])%1234567;
        }
        return arr[n];
    }
}

File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> arr;

int solution(int n) {
    arr.push_back(0);
    arr.push_back(1);
    for(int i =2; i<=n; i++){
        arr.push_back((arr[i-1]+arr[i-2])%1234567);
    }
    return arr[n];
}